src/server.js

```diff
@@ -357,6 +358,18 @@ export class Server extends EventEmitter {
         .nodeify(callback);
     }
 
+
+    /**
+     * Calls handler by ID and reads it's usage statistics.
+     * @return {Object} statistics { srcTxBytes, srcRxBytes, trgTxBytes, trgRxBytes }
+     */
+    getStatisticsForHandler(handlerId) {
```

Comment:
Let's rename "Statistics" to "Stats", it's easier to write :)

Reply:
And maybe I'd rename this to getConnectionStats(connectionId), it will make it more clear for users.